nomad/structs/structs.go

```diff
@@ -5553,6 +5569,37 @@ func (t *Task) Warnings() error {
 	return mErr.ErrorOrNil()
 }
 
+type Kind string
+
+const connect_prefix = "connect:"
```

Comment:
`connectPrefix` is more a more go-ish name.

Reply:
Just realized I'll need to reference this elsewhere, so `ConsulConnectPrefix` is ideal.

I can always do that myself though.